selenium_tests/base_test_case.py

```diff
@@ -0,0 +1,71 @@
+import unittest
+import xlrd
+from pyvirtualdisplay import Display
+from django.conf import settings
+
+from selenium import webdriver
+from selenium.webdriver.common.desired_capabilities import DesiredCapabilities
+
+
+# Enabling stdout logging only for high `-v`
+class BaseSeleniumTestCase(unittest.TestCase):
+    driver = None  # make selenium driver available to any part of the test case
+    display = None  # a reference to the virtual display (for running tests locally)
+
+    @classmethod
+    def setUpClass(cls):
+        """
+        Sets up the test case, including the selenium browser driver to use
+        """
+        # todo: tlt-1263 local/remote should be configurable from outside the code
+
+        local = False
```

Comment:
We can probably make this configurable from within our settings. Let's discuss it when you've gone over the PR changes.